examples/nlp/text_normalization_as_tagging/conf/thutmose_tagger_itn_config.yaml

```diff
@@ -0,0 +1,115 @@
+name: &name itn
+lang: ???        # e.g. 'ru', 'en'
+
+# Pretrained Nemo Models
+pretrained_model: null
+
+trainer:
+  devices: 1 # the number of gpus, 0 for CPU
+  num_nodes: 1
+  max_epochs: 3  # the number of training epochs
+  checkpoint_callback: false  # provided by exp_manager
+  logger: false  # provided by exp_manager
+  accumulate_grad_batches: 1 # accumulates grads every k batches
+  gradient_clip_val: 0.0
+  precision: 32 # Should be set to 16 for O1 and O2 to enable the AMP.
+  accelerator: gpu
+  strategy: ddp
+  log_every_n_steps: 1  # Interval of logging.
+  val_check_interval: 1.0  # Set to 0.25 to check 4 times per epoch, or an int for number of iterations
+  resume_from_checkpoint: null # The path to a checkpoint file to continue the training, restores the whole state including the epoch, step, LR schedulers, apex, etc.
+
+model:
+  do_training: true
+  label_map: ./label_map.txt
+  semiotic_classes: ./semiotic_classes.txt
```

Comment:
if label_map and semiotic_classes are provided by a user, they should have `???` values, i.e. required